03.bowling/bowling.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = shots.each_slice(2).map(&:itself)
```

Comment:
あ、すいません、前回はmapをオススメしましたが、ここはこっちの方がシンプルですね。

```suggestion
frames = shots.each_slice(2).to_a
```